src/m_misc.c

```diff
@@ -2344,6 +2344,13 @@ default_t defaults[] = {
     "1 to force integer scales"
   },
 
+  {
+    "vga_porch_flash",
+    (config_t *) &vga_porch_flash, NULL,
+    {0}, {0, 1}, number, ss_none, wad_no,
```

Comment:
This feels so natural to me, that I seriously consider to enable it by default.

Reply:
I find it a bit odd - it's only visible in 4:3 mode and there are these huge black bars that we didn't have on real monitors back then. I like it as a secret feature 😉